### ISSUE TITLE
Move upload wizard's location picker to its own step if the pic has no location

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#4454]
After #8 is implemented and released, it might be worth making it more visible when needed.

A picture with a location are much much more valuable than the same picture without a location.

So, if the pic has no location (and none has been selected via the button in the caption/description step), I would suggest forcing the location picker to appear. It should have an easy-to-find and easy-to-use way to skip, so that people are not tempted to enter erroneous coordinates.

The same screen could contain the following message: "_Tip: Set your camera to record location. [Learn how](https://commons-app.github.io/docs.html#geotagging)_"

**Comment by Ayan-10** [comment]
@nicolas-raoul Can you little elaborate me about this? As I am working on #8 I can do this along with it.

**Comment by nicolas-raoul** [comment]
Duplicate of  #1848
